build.sbt

```diff
@@ -67,11 +67,23 @@ ThisBuild / developers := List(
 
 ThisBuild / githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_))
 ThisBuild / tlCiReleaseBranches := Seq("series/0.7")
-ThisBuild / tlSitePublishBranch := Some("series/0.7")
+// ThisBuild / tlSitePublishBranch := Some("series/0.7")
+ThisBuild / tlSitePublishBranch := Some("doc/versioned-0.7")
 
 lazy val docsSettings =
   Seq(
-    fork := true,
+    tlSiteHeliumConfig ~= {
+      import laika.rewrite._
+      _.site.versions(
+        Versions(
+          currentVersion = Version("0.7.x", "0.7"),
+          olderVersions = Seq(
+            Version("0.5.x", "0.5.0"),
+            Version("0.4.x", "0.4.0")
+          )
+        )
+      )
+    },
```

Comment:
This is the version config. Any versioned branch needs to include its `currentVersion`, but the `olderVersions` stuff only needs to be kept up-to-date on `main`.

Also, the old versions specifically point to `0.5.0` instead of `0.5` because historically this is how the old site was setup.